crates/bevy_macro_utils/src/bevy_manifest.rs

```diff
@@ -1,17 +1,21 @@
 extern crate proc_macro;
 
 use proc_macro::TokenStream;
-use std::{env, path::PathBuf};
+use std::{env, path::PathBuf, sync::LazyLock};
 use toml_edit::{DocumentMut, Item};
 
 /// The path to the `Cargo.toml` file for the Bevy project.
 pub struct BevyManifest {
     manifest: DocumentMut,
 }
 
-impl Default for BevyManifest {
-    fn default() -> Self {
-        Self {
+const BEVY: &str = "bevy";
+const BEVY_INTERNAL: &str = "bevy_internal";
+
+impl BevyManifest {
+    /// Returns a global shared instance of the [`BevyManifest`] struct.
+    pub fn default_shared() -> &'static LazyLock<Self> {
```

Comment:
```suggestion
    pub fn shared() -> &'static LazyLock<Self> {
```
and all the places where it's called too.

It was called `default` because of the `Default` impl, but it doesn't make sense to keep that part of the name

Reply:
Makes sense.